app/javascript/mastodon/components/dropdown_menu.jsx

```diff
@@ -35,6 +35,7 @@ class DropdownMenu extends React.PureComponent {
   handleDocumentClick = e => {
     if (this.node && !this.node.contains(e.target)) {
       this.props.onClose();
+      e.stopPropagation();
```

Comment:
Isn't that inconsistent with the `passive: true` option used on `touchend`?

Reply:
`passive` is about `preventDefault()`,  not `stopPropagation()`, according to MDN

Reply:
You're right, that makes sense. Looks good then.